### DISCUSSION TITLE
assets

### DISCUSSION THREAD

**Discussion by chayakomar** [pimcore/pimcore#12967] [Ideas]
### Feature description

While uploading an asset of a picture (jpg, png) there's options of downloading formats such as web, office, original. 
can there be an option of downloading pdf?
thank you

**Comment by dvesh3** [comment]
@chayakomar we always use the format `jpeg` here https://github.com/pimcore/pimcore/blob/a0ca76adc00891a8aeb50d93b6123730090928ef/bundles/AdminBundle/Controller/Admin/Asset/AssetController.php#L1201 for these options. It would be great if you could provide a PR to change the format based on options.
